hexrd/findorientations.py

```diff
@@ -155,7 +152,6 @@ def generate_orientation_fibers(cfg, eta_ome):
                     discretefiber_reduced, input_p, chunksize=chunksize
                 ), total=ntotal
             ):
-            pass
```

Comment:
The commented out code should probably all be removed, rather than just the pass statement.